examples/tv-casting-app/linux/main.cpp

```diff
@@ -106,6 +106,7 @@ CHIP_ERROR ProcessClusterCommand(int argc, char ** argv)
 
 int main(int argc, char * argv[])
 {
+    ChipLogProgress(AppServer, "chip_casting_simplified = 0");
```

Comment:
These logs seem hard-coded and tied to a BUILD.gn flag (which is a bool).
There seem to be no direct linkage between the gn flag and the log here ... we need something else probably, otherwise this may be a lie. This seems to just say "chip_casting_simplified = 0" whenever I run main.cpp .... they are not tied.

Reply:
I did not pull the value of chip\_casting\_simplified to log in these files, because as per the BUILD.gn, these files are only built/run if chip\_casting\_simplified is set to 0 or 1, respectively for main.cpp and simple-app.cpp. See https://github.com/project-chip/connectedhomeip/blob/master/examples/tv-casting-app/linux/BUILD.gn#L27-L39

I've updated the PR summary to call out this change and also added a comment inline in the code to state the above.